src/components/EventCard/EventCard.js

```diff
@@ -0,0 +1,95 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
+import styles from './EventCard.module.css';
+import { Button } from 'react-bootstrap';
+import React from "react";
+import Image from 'next/image'
+
+function EventCard( {title, image, author, authorPic, maxCap, currCap, location, time, description, joined, owner} ) {
+    EventCard.propTypes = {
+        title: String,
+        image: String,
+        author: String,
+        authorPic: String,
+        maxCap: Number,
+        currCap: Number,
+        location: String,
+        time: String,
+        description: String,
+        joined: Boolean,
+        owner: Boolean,
+    }
+    let buttText;
+    if (joined) {
+        buttText = "Happening Soon!";
+    } else {
+        buttText = "Join!"
+    }
+    let badge;
+    if (owner) {
+        badge = <OwnerBadge />
+    } else {
+        badge = <div />
+    }
```

Comment:
add spaces between logic blocks so easier to read

- space between let buttText
- space before let badge

Reply:
Fixed

<img width="443" alt="image" src="https://user-images.githubusercontent.com/112282141/227413658-ac05c7f3-89c5-4b24-a919-ac447492927a.png">